### PR TITLE
feat: add bank-account-list and refresh

### PR DESCRIPTION
**Pull request by dcocking**
### Type of change

- [ ] Chore
- [X] Feature
- [ ] Bug fix

### Linked issue
https://cybrid.atlassian.net/browse/CYB-842

### Why
The customer needs to be able to view a list of linked external bank accounts, as well as reconnect accounts with the state `refresh_required`

### How
By adding a `bank-account-list` component, and adding refresh functionality to the `bank-account-connect` component.

Screen Shot 2023-01-31 at 11 07 39 PM

Screen Shot 2023-01-31 at 11 08 11 PM

Screen Shot 2023-01-31 at 11 08 06 PM

### CODE REVIEW

File: library/src/environments/environment.prod.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 export const environment = {
   production: true,
   demoBankApiBasePath: 'https://bank.demo.cybrid.app',
+  localBankApiBasePath: 'http://api-platform-bank.local.cybrid.com:3002',
```
Comment (brent-cybrid) [inline library/src/environments/environment.prod.ts:4]: Should this be in this PR?

**Review by brent-cybrid — APPROVED** [review]
How are we handling the case where the link token is not returned during the polling interval?

File: src/environments/environment.prod.ts [hunk 2]
```diff
@@ -8,12 +8,14 @@ export const environment = {
   production: true,
```
[the remark this answers is not in the document]
Comment (dcocking) [reply src/environments/environment.prod.ts:8]: These changes were missed when I added local support.

**Comment by dcocking** [comment]
> How are we handling the case where the link token is not returned during the polling interval?

Reconnect uses the same bank-account-connect component. If a link token is not returned within the polling duration (currently set at 5000ms) it triggers a timeout and then the UI error 'unexpected error'.

**Pull request merged**